modules/tiles/docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
```

Comment:
You have excellent material in your RFC that you can just copy in here, plus move/copy some of the docs from 3d-tiles...